purrplingcat/PurrplingMod
# Feature/portugal translation

## Types of changes

- [x] Content updates (new dialogues added or some string edited and etc)
- [ ] **Merge ASAP**
- [ ] Merge before *enumerated issues or other PRs numbers*
- [x] Merge after #50 

## Description
Adds Portuguese Brazilian translation.

## How to test
Switch to Portuguese and check all strings and dialogues. It must be in Portuguese.

## Checklist

**Please check if the PR fulfills these requirements**
- [x] Changes was tested and passed

## Other information

Thanks to [andril11](https://www.nexusmods.com/users/68848663) for translation to Portuguese Brazilian.

---

Requires #50
